### DISCUSSION TITLE
Portal Guest Login

### DISCUSSION THREAD

**Discussion by aleksandarnikolic85** [pimcore/pimcore#12505] [Q&A / Support]
Hello everybody,

I have a problem with the Pimcore Portal Guest Login. 
Guest Login works perfectly if I type the URL in the browser. I will be able to reach the Portal as logged in portal user.

But in the case if that URL I use as an <a href=""> tag on my website, this Guest Login does not work. As I can see I am logged in, but also redirected to the regular log in page.

Summary: I can log in as a guest directly, but not if I am redirected.

Does somebody know how to solve it?
Thanks in advance

**Comment by fashxp** [comment]
Hi, 
I cannot follow you. Can you try once more to explain :-) .

P.S.: Please note, this is a community channel which depends on community responses. If you need enterprise support, please contact your contact person at Pimcore to get access. Thank you.

**Comment by Refaat-alktifan** [comment]
Hi @fashxp , maybe you can help me, after I installed the portal engine I added the configuration for guest users. 
I can now see the "Continue as guest" button in the Frontend, but when I go to auth/guest-login to use the Portal Engine as a guest, I always get a 404 error. 
Is there any configuration I should adjust to enable the portal engine guest user, to fix the 404 for for the auth/guest-login route?

https://pimcore.com/docs/portal-engine/current/Installation/index.html

https://pimcore.com/docs/portal-engine/current/Administration_of_Portals/Configuration/User_Management/Guest_User.html
